src/modules/users/users.service.ts

```diff
@@ -1,13 +1,19 @@
-import { Injectable } from '@nestjs/common';
+import { Inject, Injectable, NotFoundException } from '@nestjs/common';
+import * as bcrypt from 'bcrypt';
 import { CreateUserDto } from './dto/create-user.dto';
 import { UpdateUserDto } from './dto/update-user.dto';
 import { UsersRepository } from './users.repository';
-
+import { RolesService } from './../roles/roles.service';
+import { AssignRoleDto } from './dto/assign-role.dto';
 @Injectable()
 export class UsersService {
-  constructor(private usersRepository: UsersRepository) {}
+  constructor(
+    private usersRepository: UsersRepository,
+    @Inject(RolesService)
+    private rolesService: RolesService,
+  ) {}
 
-  create(createUserDto: CreateUserDto) {
+  async create(createUserDto: CreateUserDto) {
```

Comment:
Maybe this is a mistake, I should remove async work because I'm not using any await inside this function.